src/validation/presenter.validation.ts

```diff
@@ -1,8 +1,9 @@
 import { body } from "express-validator";
 
 import { ErrorMessages } from "./error.messages";
+import { VALID_CHARACTERS } from "./regex/regex.validation";
 
 export const presenter = [
-  body("full_name").not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.FULL_NAME).isLength({ max: 160 }).withMessage(ErrorMessages.MAX_FULL_NAME_LENGTH),
+  body("full_name").not().isEmpty({ ignore_whitespace: true }).withMessage(ErrorMessages.FULL_NAME).isLength({ max: 160 }).withMessage(ErrorMessages.MAX_FULL_NAME_LENGTH).matches(VALID_CHARACTERS).withMessage(ErrorMessages.INVALID_CHARACTERS),
```

Comment:
.matches(VALID_CHARACTERS).withMessage(ErrorMessages.INVALID_CHARACTERS)

doesn't make sense and I think the VALID_CHARACTERS constant should change its name back to INVALID_CHARACTERS.